flutter/lib/src/shared/services/store_service.dart

```diff
@@ -30,7 +30,7 @@ class StoreService {
       );
 
   Future<void> putThemeMode(String mode) async {
-    await _box.put(Keys.themeMode, mode);
+    await _box.put(Keys.themeMode.toString(), mode);
```

Comment:
This is required because the key needs to be a string. We could have used `.name` instead but `.toString()` gives a better name spacing.